omero/developers/PythonBlitzGateway.txt

```diff
@@ -1,19 +1,70 @@
 Blitz Gateway documentation
 ---------------------------
 
-The Sphinx-generated documentation of methods provided by OMERO Gateway
-is available showing :pythondoc:`wrapper classes <omero/omero.gateway.html>`.
+This page provides some background information on the OMERO Python client
+'gateway' (omero.gateway module).
+
+The Blitz Gateway is a Python client-side library that facilitates working
+with the OMERO API, handling connection to the server, loading of data objects
+and providing convenience methods to access the data. It was originally
+designed as part of the |OmeroWeb| framework, to provide connection and data
```

Comment:
Same in line 139 (not introduced by the PR but could be fixed on that occasion, too).